src/Update/Nix/FetchGit/Utils.hs

```diff
@@ -73,14 +77,10 @@ matchAttr t = \case
   NamedVar _ _ -> Nothing
   Inherit _ _  -> Nothing
 
-traverse2 :: (Traversable t, Traversable s, Applicative f)
-          => (a -> f b) -> t (s a) -> f (t (s b))
-traverse2 f = fmap getCompose . traverse f . Compose
-
-sequenceA2 :: (Traversable t, Traversable s, Applicative f)
-            => t (s (f a)) -> f (t (s a))
-sequenceA2 = fmap getCompose . sequenceA . Compose
-
-mapConcurrently2 :: (Traversable t, Traversable s) => (a -> IO b) -> t (s a) -> IO (t (s b))
-mapConcurrently2 f = fmap getCompose . mapConcurrently f . Compose
-
+-- Takes an ISO 8601 date and returns just the day portion.
+parseISO8601DateToDay :: Text -> Either Warning Day
```

Comment:
There is only [one parsing function](https://hackage.haskell.org/package/iso8601-time-0.1.4/docs/Data-Time-ISO8601.html#v:parseISO8601) in there and it converts the timestamp to UTC, throwing away any information about the time zone.  This means that the date we get could be different than the date displayed in `git log`, which would cause confusion.  The [nixpkgs manual](http://nixos.org/nixpkgs/manual/) does not address the issue of timezones; it just says:

> If package is fetched from git's commit then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also add "git" to the name - e.g., "pkgname-git-2014-09-23".